### PR TITLE
Don't log to debug for every failed extrinsic. 

### PR DESCRIPTION
**Pull request by kianenigma**
Another step toward cleaning the runtime logs.

### CODE REVIEW

File: frame/system/src/lib.rs [hunk 1]
```diff
@@ -1449,7 +1449,12 @@ impl<T: Config> Module<T> {
 			match r {
 				Ok(_) => Event::ExtrinsicSuccess(info),
 				Err(err) => {
-					sp_runtime::print(err);
+					log::trace!(
+						target: "runtime::system", 
+						"failed extrinsic at {:?} => {:?}",
```
Comment (bkchr) [inline frame/system/src/lib.rs:1454]:
```suggestion
						"Extrinsic failed at block({}): {:?}", 
```

**Review by bkchr — APPROVED** [review]

File: frame/system/src/lib.rs [hunk 2]
```diff
@@ -1449,7 +1449,12 @@ impl<T: Config> Module<T> {
 			match r {
 				Ok(_) => Event::ExtrinsicSuccess(info),
 				Err(err) => {
-					sp_runtime::print(err);
+					log::trace!(
```
Comment (xlc) [inline frame/system/src/lib.rs:1452]: can we make it debug?
```suggestion
					log::debug!(
```

**Review by shawntabrizi — APPROVED** [review]

File: frame/system/src/lib.rs (hunk 2 above)
Comment (bkchr) [reply frame/system/src/lib.rs:1452]: The point of this pr is to not to be use debug.

Comment (xlc) [reply frame/system/src/lib.rs:1452]: It used to be unconditionally print, default log level is info. For failed extrinsic, I think it is fair to make it higher verbose level than trace?

Comment (xlc) [reply frame/system/src/lib.rs:1452]: Right. I thought sp_runtime::print is info level (I think it used to be?).

Comment (bkchr) [reply frame/system/src/lib.rs:1452]: No. It was just `println!` back in the days, until we changed it to `log::debug!`

**Comment by kianenigma** [comment]
bot merge

File: frame/system/src/lib.rs [hunk 3]
```diff
@@ -1449,7 +1449,12 @@ impl<T: Config> Module<T> {
...
+						target: "runtime::system", 
+						"Extrinsic failed at block({}): {:?}",
```
Comment (kianenigma) [inline frame/system/src/lib.rs:1454]:
```suggestion
						"Extrinsic failed at block({:?}): {:?}", 
```

**Pull request merged**